Core/include/Acts/Propagator/EigenStepper.ipp

```diff
@@ -133,9 +133,7 @@ Acts::Result<double> Acts::EigenStepper<B, E, A>::step(
         h2 * ((sd.k1 - sd.k2 - sd.k3 + sd.k4).template lpNorm<1>() +
               std::abs(sd.kQoP[0] - sd.kQoP[1] - sd.kQoP[2] + sd.kQoP[3])),
         1e-20);
-    return (error_estimate <= state.options.tolerance) &&
-           ((h.currentType() != ConstrainedStep::accuracy) ||
-            (error_estimate >= state.options.tolerance / 10));
```

Comment:
We can follow up in another issue why this is a problem for the Fatras simulation but for the moment I will merge this is.